### PR TITLE
MQTT certs extraction on getting started guide

### PR DESCRIPTION
**Pull request by ppatierno**
Used oc extract command for extracting MQTT certs
Updated MQTT receiver and sender command lines
Removed useless MQTT receiver command line

### CODE REVIEW

**Comment by lulf** [comment]
@ppatierno This command will not work for kubernetes, so this is why I used this way for extractions.

File: documentation/getting_started/mqtt-client-openshift.adoc [hunk 1]
```diff
@@ -166,17 +166,11 @@ client.connect(opts.connectHost, opts.portHost, 60)
 client.loop_forever()
 ----
 
-
-[options="nowrap",subs=attributes+]
-....
-./tls_mqtt_recv.py -c "$({OcGetRoute} mqtt)" -p 443 -t mytopic -q 1 -s ./server-cert.pem
-....
-
```
Comment (ppatierno) [inline documentation/getting_started/mqtt-client-openshift.adoc]: @lulf I got the reason for not using the oc extract so I was missing the reason. Thanks ! But what about this receiver line ... is it just a mistake for being here ?

Comment (ppatierno) [reply documentation/getting_started/mqtt-client-openshift.adoc]: @lulf Can I close this PR and get rid of this lines in the master ?

**Comment by ppatierno** [comment]
I'm goint to close this and just removing the useless receiver command line on the master

**Pull request closed without merging**